README.md

```diff
@@ -29,7 +29,7 @@ Contact Form Plugin
 1. Run `$ composer require mangoweb-sylius/sylius-contact-form-plugin`.
 2. Register `\MangoSylius\ContactFormPlugin\MangoSyliusContactFormPlugin` in your Kernel.
 3. Import `@MangoSyliusContactFormPlugin/Resources/config/routing.yml` in the routing.yml.
-4. Import `@MangoSyliusContactFormPlugin/Resources/config/resources.yml` in _sylius.yml.
+4. Import `@MangoSyliusContactFormPlugin/Resources/config/resources.yml` and `@MangoSyliusContactFormPlugin/Resources/config/mailer.yml` in _sylius.yml.
```

Comment:
Create one file `config.yml` in this file include `resources.yml` and `mailer.yml`.
So that the user has to take as few steps as possible.

Reply:
DONE

Reply:
include resources and mailer into config, no copy into file :)

Reply:
DONE

Reply:
I don't understand, it's still copied there

Reply:
DONE